### PR TITLE
Update CVE-2019-6338.json

### PR DESCRIPTION
**Pull request by mlhess**

### CODE REVIEW

File: 2019/6xxx/CVE-2019-6338.json [hunk 1]
```diff
@@ -1,18 +1,106 @@
...
+        }
       ]
-   }
+    }
+  },
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "In Drupal Core  versions 7.x prior to 7.6.2, 8.6.x prior to 8.6.6 and 8.5.x prior to 8.5.9; Drupal core uses the third-party PEAR Archive_Tar library. This library has released a security update which impacts some Drupal configurations. Refer to CVE-2018-1000888 for details"
```
Comment (cve-team) [inline 2019/6xxx/CVE-2019-6338.json]: The fix for the 7.x release should be 7.62, not 7.6.2, shouldn't it? The former is what appears in the "affects" container above and in SA-CORE-2019-001.

**Comment by mlhess** [comment]
Yes, can we update the json tool to populate that text automagicly?

**Comment by cve-team** [comment]
Are you referring to Vulnogram or some other json tool?

Also, I'm not clear what you mean by "populate that text automatigicly". Can you explain or give an example?

**Comment by mlhess** [comment]
Yes, if we fill in the versions for that, it could pre-populate the description field with the versions that are filled in.  A user could always delete it, but it might make doing these a tad easier.

**Comment by cve-team** [comment]
MITRE doesn't maintain Vulnogram unfortunately. You could ask the maintainer (Chandan B N) to add that functionality, say, by creating an issue (https://github.com/Vulnogram/Vulnogram/issues).

**Pull request merged**